pkg/nsx/services/firewall.go

```diff
@@ -37,8 +38,6 @@ type SecurityPolicyService struct {
 	RuleStore           cache.Indexer
 }
 
-var log = logf.Log.WithName("service").WithName("firewall")
-
 // InitializeSecurityPolicy sync NSX resources
 func InitializeSecurityPolicy(NSXClient *nsx.Client, cf *config.NSXOperatorConfig) (*SecurityPolicyService, error) {
```

Comment:
We move this InitiallizeSecurityPolicy to store.go in v4.0.0 and v3.2.1 for this patch. However, I agreed with Qian, this func is better to put in firewall.go instead of store.go.
please do the same change in v4.0.0 and v3.2.1.

Reply:
ok